DotNetExcel/StarkBankExcel/GetStatement.Designer.cs

```diff
@@ -154,7 +154,7 @@ public sealed partial class GetStatement : Microsoft.Office.Tools.Excel.Workshee
             this.button1.BackColor = System.Drawing.SystemColors.Control;
             this.button1.ForeColor = System.Drawing.SystemColors.ControlText;
             this.button1.Name = "button1";
-            this.button1.Text = "Baixar Extrato";
+            this.button1.Text = "Extrato";
```

Comment:
Esse botão faz o GET do extrato ou faz o download do extrato para a maquina do cliente?
Se a função for download, acredito que faça sentido manter o texto "baixar"